.eslintrc.json

```diff
@@ -0,0 +1,94 @@
+{
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "node": true
+  },
+  "globals": {
+    "describe": true,
+    "expect": true,
+    "it": true,
+    "spyOn": true,
+    "App": true,
+    "Backbone": true,
+    "i18n": true,
+    "xdescribe": true,
+    "xit": true,
+    "it": true,
+    "beforeEach": true,
+    "beforeAll": true,
+    "afterEach": true,
+    "sinon": true,
+    "jasmine": true,
+    "loadFixtures": true,
+    "setFixtures": true,
+    "testEventHandler": true,
+    "Properties": true,
+    "Factory": true,
+    "_": true,
+    "getJSONFixture": true,
+    "google": true,
+    "Handlebars": true,
+    "NProgress": true,
+    "Marionette": true,
+    "monaco": true,
+    "ZeroClipboard": true,
+    "AsyncSpec": true,
+    "moment": true,
+    "md5": true,
+    "printStackTrace": true,
+    "Notifier": true,
+    "Showdown": true,
+    "async": true,
+    "Formbuilder": true,
+    "$fh": true,
+    "logger": true,
+    "semver": true,
+    "baseSuperListSetup": true,
+    "baseSuperListSearchAssertions": true,
+    "baseSuperListAssertions": true,
+    "prettyPrint": true,
+    "jsoneditor": true,
+    "Notification": true,
+    "destroyGlobals": true,
+    "mockGlobals": true,
+    "fauxServer": true,
+    "c3": true,
+    "Datamap": true,
+    "d3": true,
+    "angular": true,
+    "$": true,
+    "jQuery": true,
+    "xssFilters": true
+  },
+  "extends": "eslint:recommended",
+  "rules": {
+    // TODO remove line "no-unused-vars"
```

Comment:
It is working as well and these are the points that we need work in the other tasks to finish it. Do you think is better remove the TODO?